README.md

```diff
@@ -1 +1,175 @@
-# module1microverse
\ No newline at end of file
+<a name="readme-top"></a>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+# 📖 [Setup and Mobile First Project] <a name="about-project"></a>
+
+> This is my first Microverse Project. It contains some things I am capable of doing as a software developer.
+
+**[HELLO MICROVERSE PROJECT]** is a responsive website that tests my web development skills. 
+
+## 🛠 Built With <a name="HTML and CSS"></a>
+
+### Tech Stack <a name="Front end"></a>
+
+> This is a Mobile First website.
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://html.org/">HTML</a></li>
+  </ul>
+  <ul>
+    <li><a href="https://css.org/">CSS</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+> Describe between 1-3 key features of the application.
```

Comment:
- [ ] Please remove all these comments that begin with `>` as they were placed in this README file as a guide for you and should have been removed after editing this file to your test.